### PR TITLE
Document using multiple commands in hooks

### PR DESCRIPTION
**Pull request by nrb**
Signed-off-by: Nolan Brubaker <brubakern@vmware.com>

### CODE REVIEW

File: site/docs/v1.3.1/hooks.md [hunk 1]
```diff
@@ -75,7 +75,15 @@ velero backup get nginx-hook-test
...
+To use multiple commands, wrap your target command in a shell and separate them with `;`, `&&`, or other shell conditional constructs.
+
+```shell
+    pre.hook.backup.velero.io/command='["/bin/bash", "-c", "echo hello > hello.txt && echo goodbye > goodbye.txt"]'
+```
+
 
 [1]: api-types/backup.md
-[2]: https://github.com/vmware-tanzu/velero/blob/v1.3.1/examples/nginx-app/with-pv.yaml
+[2]: https://github.com/vmware-tanzu/velero/blob/master/examples/nginx-app/with-pv.yaml
```
Comment (ashish-amarnath) [inline site/docs/v1.3.1/hooks.md]: Is this change intentional?
We might want to keep the examples also versioned, just in case we update this example in master.

Comment (nrb) [reply site/docs/v1.3.1/hooks.md]: It was not...I just copied the file directly over, will undo it.

**Comment by nrb** [comment]
@carlisia @ashish-amarnath Ok, updated!
